### ISSUE TITLE
[A11Y] - Missing aria label on overflow menu toggle

### ISSUE THREAD

**Issue by josephcoombes** [Uniswap/v3-info#217]
**Bug Description**
This button (3 dots) doesn't have any text in it, so when a screenreader is reading it, it only speaks the word "Button". Users with screenreaders will have no idea what this button does. 

<img width="263" alt="image" src="https://user-images.githubusercontent.com/6978295/162487399-7a5edc14-8c79-450d-afde-dc849c61e319.png">

**Steps to Reproduce**
If you have a screenreader installed, you can navigate to this item and should notice that the screenreader says "button".

You can install a screenreader here: https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn/related?hl=en or enable on your OS. 

**Expected Behavior**
This should have a descriptive label `aria-label`. At the very least, it should say something like "Menu", but as clear and descriptive as possible is ideal.
